src/block_cache/block_chunkcache.c

```diff
@@ -249,6 +249,87 @@ __chunkcache_eviction_thread(void *arg)
     return (WT_THREAD_RET_VALUE);
 }
 
+/*
+ * __chunkcache_str_cmp --
+ *     Qsort function: sort string array.
+ */
+static int WT_CDECL
+__chunkcache_str_cmp(const void *a, const void *b)
+{
+    return (strcmp(*(const char **)a, *(const char **)b));
+}
+
+/*
+ * __chunkcache_arr_free --
+ *     Free the array of strings.
+ */
+static void
+__chunkcache_arr_free(WT_SESSION_IMPL *session, char ***arr)
+{
+    char **p;
+
+    if ((p = (*arr)) != NULL) {
+        for (; *p != NULL; ++p)
+            __wt_free(session, *p);
+        __wt_free(session, *arr);
+    }
+}
+
+/*
+ * __config_gets_pinned_objects --
+ *     Get sorted array of pinned objects from the config.
+ */
+static int
```

Comment:
Nice work getting this into a different function, that's a good simplification. Is the `s` in `gets` shorted for "sorted"?

Reply:
Not really, I was just following the convention of `__wt_config_gets`. Let me know if you want me to remove the s.

Reply:
Ah I see. I believe that function is modelled after the C standard library function `gets` where `gets` is short for `get string`. In that case, I think this can simply be called `__config_get_pinned_objects`, or maybe something to show that it's sorted? e.g. `__config_sort_pinned_objects`. Not sure what the best option has, I'm happy to talk about it offline.

Reply:
Updated in [dccdecc](https://github.com/wiredtiger/wiredtiger/pull/9438/commits/dccdecc42fb0241ad29f258f9e1448831ba61d7b)